src/Compilers/CSharp/Test/Syntax/Parsing/CSharpParseOptionsTests.cs

```diff
@@ -73,5 +52,189 @@ public void TestFieldsForEqualsAndGetHashCode()
                 "PreprocessorSymbols",
                 "SpecifiedLanguageVersion");
         }
+
+        [Fact]
+        public void SpecifiedKindIsMappedCorrectly()
+        {
+            var options = new CSharpParseOptions(kind: SourceCodeKind.Regular);
+            Assert.Equal(SourceCodeKind.Regular, options.Kind);
+            Assert.Equal(SourceCodeKind.Regular, options.SpecifiedKind);
+
+            options.Errors.Verify();
```

Comment:
>options.Errors.Verify(); [](start = 12, length = 24)

Consider testing that an instance, which is created without passing any explicit values, has no errors.